client/lib/react-player/LICENSE

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```

Comment:
Same with this file. If it was not intentional then we should probably remove these files.